src/GitHub/Endpoints/Repos/Statuses.hs

```diff
@@ -0,0 +1,43 @@
+--
+-- The repo statuses API as described on
+-- <https://developer.github.com/v3/repos/statuses/>.
+module GitHub.Endpoints.Repos.Statuses (
+    createStatus,
+    createStatusR,
+    statusesFor,
+    statusesForR,
+    statusFor,
+    statusForR,
+    module GitHub.Data
+    ) where
+
+import GitHub.Data
+import GitHub.Internal.Prelude
+import GitHub.Request
+import Prelude ()
+
+createStatus :: Auth -> Name Owner -> Name Repo -> Name Commit -> NewStatus -> IO (Either Error Status)
+createStatus auth owner repo sha ns =
+    executeRequest auth $ createStatusR owner repo sha ns
+
```

Comment:
could you add links to the relevant parts of github docs here as haddocks, and to `Github.hs` module explicitly, so we get a nice overview of api: http://hackage.haskell.org/package/github-0.15.0/docs/GitHub.html